app/models/gravatar.rb

```diff
@@ -0,0 +1,36 @@
+class Gravatar
+  attr_reader :email
+
+  def initialize(email, options = {})
+    raise ArgumentError, "Expected :email" unless email
```

Comment:
We can remove the condition by making `email` a required kwarg. We can rely on the parser to ensure argument correctness:

```ruby
def initialize(email:)
  @email = email
end
```